### PR TITLE
Much better rewrite, generate install_manifest.text with scons

### PR DESCRIPTION
**Pull request by ryanthecoder**
Take advantage of an ENV variable we already had to create the manifest file

BST-263
http://jira.makerbot.net/browse/BST-263

### CODE REVIEW

File: mb_install.py [hunk 1]
```diff
@@ -144,6 +144,9 @@ def mb_install_system(env, source, dest):
     return target
 
 def mb_create_install_target(env):
+    with open('install_manifest.txt', 'w') as fp:
```
Comment (tedbrandston) [inline mb_install.py]: `env.File('#/install_manifest.txt').abspath`

Comment (dudecc) [inline mb_install.py]: Aww, I liked that it went in obj/

Comment (tedbrandston) [inline mb_install.py]: hmm, I suppose you're right. Ryan was saying there was some problem with it being in obj when trying to access it. Also, obj only exists on some platforms.

@ryanthecoder what was wrong with obj?

Comment (ryanthecoder) [inline mb_install.py]: The Python based repositories don't have obj directories on any platform

**Pull request closed without merging**